.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: dart
+dart:
+  - dev
```

Comment:
I'm happy to add anything you like, I used the named versions to avoid things going stale. :/  Doesn't "dev" cover 1.22?